server/db/migrate/20150429180751_add_root_passwords_to_deployments.rb

```diff
@@ -0,0 +1,6 @@
+class AddRootPasswordsToDeployments < ActiveRecord::Migration
+  def change
+    add_column :fusor_deployments, :rhev_root_password, :string
+    add_column :fusor_deployments, :cfme_root_password, :string
```

Comment:
@isratrade, Understood and it is is good to ask.  I can see the future providing some additional flexibility.